src/cpp/session/session-options.json

```diff
@@ -719,13 +719,6 @@
             "defaultValue": {"code": "kDefaultRsclangPath", "description": "bin/rsclang"},
             "description": "Specifies the path to the libclang shared library"
          },
-         {
-            "name": "external-libclang-headers-path",
```

Comment:
Will this cause problems (like session failing to start) if someone had this setting in rsession.conf? Maybe we should leave this in and mark it as `isDeprecated`, such as seen here?

https://github.com/rstudio/rstudio/blob/7781d230ce5abdcd20d185b385f1a0ee01df6da8/src/cpp/session/session-options.json#L575-L581

Reply:
Good point; I've added it back in. Better safe than sorry.